Monika After Story/game/script-moods.rpy

```diff
@@ -882,3 +882,27 @@ label mas_mood_excited:
             m 3eua "I know I'm always excited to see you every day."
             m 1hub "Either way, I'm glad that you're happy!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_burnt_out",
+            prompt="...burnt out.",
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_burnt:
+    m 1wud "Oh!"
+    m 1ekd "I'm so sorry to hear you feel that way, [player]..."
+    m 2esc "But, if that's the case..."
```

Comment:
When using `extend` like this, you must make sure the line will automatically move into the next
```suggestion
    m 2esc "But, if that's the case...{w=0.3}{nw}"
```